File: CS/DXSample/ViewModel/TabViewModel.cs
using DevExpress.Mvvm.DataAnnotations;
using System;

namespace DXSample.ViewModel {
    [POCOViewModel]
    public class TabViewModel {
        public virtual DateTime Updated { get; set; }
        public virtual object Content { get; set; }

        public TabViewModel() {
            Updated = DateTime.Now;
            Content = Guid.NewGuid();

        }
        public void UpdateDocument() {
            Updated = DateTime.Now;
            Content = Guid.NewGuid();
        }
    }
}

File: VB/DXSample/ViewModel/TabViewModel.vb
Imports DevExpress.Mvvm.DataAnnotations
Imports System

Namespace DXSample.ViewModel

    <POCOViewModel>
    Public Class TabViewModel

        Public Overridable Property Updated As Date

        Public Overridable Property Content As Object

        Public Sub New()
            Updated = Date.Now
            Content = Guid.NewGuid()
        End Sub

        Public Sub UpdateDocument()
            Updated = Date.Now
            Content = Guid.NewGuid()
        End Sub
    End Class
End Namespace
